pgp-keys-override.list

```diff
@@ -0,0 +1,230 @@
+# syntax:
+# groupId:artifactId:version =  *,any,badSig,noKey,noSig
+# wildcards * are allowed in artifactId/version
+
+# We have not yet figured out from where this is pulled :(
+io.smallrye:jandex:3.2.0 = badSig
+io.smallrye:jandex:3.2.0:pom  = badSig
```

Comment:
What do you mean by that comment?
https://mvnrepository.com/artifact/io.smallrye/jandex/3.2.0 <- this is the maven repository related to the artifact. The artifact is pulled by hibernate-core https://mvnrepository.com/artifact/io.smallrye/jandex/usages